site/frontend/src/pages/dashboard.ts

```diff
@@ -80,6 +80,12 @@ function render(
 }
 
 function renderRuntime(element: string, data: [number], versions: [string]) {
+  // Remove null and convert nanoseconds to miliseconds
+  const formattedData = data
+    .filter((data) => data != null)
+    .map((data) => data / 1_000_000);
+  const nullCount = data.length - formattedData.length;
```

Comment:
This assumes that nulls be only at the beginning, but that should be fine in our case. It would be nice to add a comment to explain this invariant here though.